addons/custom-menu-bar/addon.json

```diff
@@ -0,0 +1,133 @@
+{
+  "name": "Custom menu bar",
+  "description": "Hide specific editor menu bar items.",
+  "credits": [
+    {
+      "name": "philipp2007",
+      "link": "https://ggigabyte.repl.co/",
+      "note": "update, menus, compact user dropdown",
+      "id": "update"
+    },
+    {
+      "name": "CST1229",
+      "link": "https://github.com/CST1229/",
+      "note": "compact user dropdown",
+      "id": "dropdown"
+    },
+    {
+      "name": "lisa_wolfgang",
+      "link": "https://scratch.mit.edu/users/lisa_wolfgang/",
+      "note": "hide tutorials button",
+      "id": "tutorials-button"
+    }
+  ],
+  "tags": ["editor", "editorMenuBar", "featured"],
+  "settings": [
+    {
+      "name": "Menus",
+      "id": "menu-labels",
+      "type": "select",
+      "potentialValues": [
+        {
+          "name": "Show icons and text",
+          "id": "both"
+        },
+        {
+          "name": "Only show icons",
+          "id": "icons"
+        }
+      ],
+      "default": "both"
+    },
+    {
+      "name": "Tutorials button",
```

Comment:
I think this setting should be a show/hide boolean and follow the the other setting when shown.

Reply:
> I think this setting should be a show/hide boolean and follow the the other setting when shown.

Wdym?